lib/options-handler.js

```diff
@@ -215,12 +216,22 @@ class ClientOptionsHandler {
         });
 
         this.options.compilers = copiedCompilers;
+        this._updateOptionsHash();
+    }
+
+    _updateOptionsHash() {
+        this.optionsHash = getHash(this.options, "Options Hash V1");
+        logger.info(`OPTIONS HASH: ${this.optionsHash}`);
```

Comment:
Thanks for doing this: I don't know if we log enough stuff, compared to the amount I log in my day job's stuff we barely have any debugging trail!